Bottom: 2718ed40b0f4e9f3891f2d2f615a24ce0cfccc53
Top:    e30bc812599c35420e020f2e9e74fbb8d9aa90bf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 23:58:46 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
